motor-control/robot.py

```diff
@@ -1,37 +1,42 @@
 #!/usr/bin/env python3
 """
-    This sample program shows how to control a motor using a joystick. In the
-    operator control part of the program, the joystick is read and the value
-    is written to the motor.
-
-    Joystick analog values range from -1 to 1 and speed controller inputs also
-    range from -1 to 1 making it easy to work together. The program also delays
-    a short time in the loop to allow other threads to run. This is generally
-    a good idea, especially since the joystick values are only transmitted
-    from the Driver Station once every 20ms.
+    This sample program shows how to control a motor using a joystick. In the operator control part
+    of the program, the joystick is read and the value is written to the motor.
+
+    <p>Joystick analog values range from -1 to 1 and motor controller inputs also range from -1 to 1
+    making it easy to work together.
+
+    <p>In addition, the encoder value of an encoder connected to ports 0 and 1 is consistently sent
```

Comment:
Javadoc HTML tags shouldn't end up in the Python examples.

```suggestion
    Joystick analog values range from -1 to 1 and motor controller inputs also range from -1 to 1
    making it easy to work together.

    In addition, the encoder value of an encoder connected to ports 0 and 1 is consistently sent
```